### ISSUE TITLE
alternc-ssl broken

### ISSUE THREAD

**Issue by ulvida** [AlternC/AlternC#301]
I try a fresh install of the latest AlternC version on stretch. I follow the official manual installation, but with the repos: http://stable-3-3.nightly.alternc.org/latest/

After mysql server install, the installation of alternc, alternc-api and alternc-ssl, as documented, fails because of an unsolvable  dpendency between alternc and alternc-ssl. 

¿How should I install alternc pre-release on stretch?

**Comment by vincib** [comment]
alternc-ssl will disappear and be merged into alternc core as of 3.5
as of now, the only testable version of AlternC 3.5 is in the branch [update-domaines-php](/AlternC/AlternC/tree/feature-updatedomains-php)

**Comment by lelutin** [comment]
we will have to verify this once the rollback to bring back alternc-ssl (#416) is done.

If this is an issue, we'll have to address it. or we'll be able to close it for good if we can't reproduce.

**Comment by camlafit** [comment]
Hello

I close this issue. Alternc-ssl is no more provided with 3.5 version.
Normally with rc1 and rc2 we have solved all majors issues about this behavior
